airflow/api_connexion/security.py

```diff
@@ -42,6 +42,7 @@ def requires_access(permissions: Optional[Sequence[Tuple[str, str]]] = None) ->
     def requires_access_decorator(func: T):
         @wraps(func)
         def decorated(*args, **kwargs):
+            g.login_from_api = True
```

Comment:
Can you add a one-line comment explaining why this is needed? The relationship between this decorator and the `DefaultSessionInterface` class is not obvious.